README.md

```diff
@@ -63,15 +63,24 @@ librosa (0.x.x, /path/to/librosa)
 
 ### Hints for the Installation
 
-#### audioread
+`librosa` uses `soundfile` and `audioread` to load audio files.
+Note that `soundfile` does not currently support MP3, which will cause librosa to
+fall back on the `audioread` library.
 
-*Note that `audioread` needs at least one of the programs to work properly.*
+#### soundfile
+
+If you're using `conda` to install librosa, then all audio coding dependencies (except MP3) will be handled automatically.
+
+If you're using `pip` on a Linux environment, you may need to install `libsndfile`
+manually.  Please refer to the [SoundFile installation documentation](https://pysoundfile.readthedocs.io/#installation) for details.
 
-`librosa` uses `audioread` to load audio files.
+#### audioread and MP3 support
 
-To fuel `audioread` with more audio-decoding power (e. g. for reading MP3 files),
+To fuel `audioread` with more audio-decoding power (e.g., for reading MP3 files),
 you can either install *ffmpeg* or *GStreamer*.
 
+*Note that `audioread` needs at least one of the programs to work properly.*
+
 If you are using Anaconda, install *ffmpeg* by calling
```

Comment:
actually on osx it also works without anything installed using the system libraries for decoding. So we might rephrase "at least one program"

Reply:
Interesting -- I'll reword that to be platform-dependent.